.github/workflows/container.yml

```diff
@@ -0,0 +1,114 @@
+name: Docker Security Checks
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Code
+        uses: actions/checkout@v2
+      
```

Comment:
Does this actually run the jobs in the application container(s)?

Reply:
Ahhh so I did not understand what this was doing at first. It seems to be doing static code analysis, scanning relevant files for particular docker security issues.

My only comment is that these are pretty basic checks that could be made more helpful with some tweaking. But for now, let's merge it and we can continue to improve on these if we have time at the end.